user_tasks/apps.py

```diff
@@ -20,4 +20,4 @@ def ready(self):
         """
         Register Celery signal handlers.
         """
-        import user_tasks.signals  # pylint: disable=unused-variable
+        import user_tasks.signals  # pylint: disable=unused-variable, import-outside-toplevel, unused-import
```

Comment:
**Reviewer Note:** I imagine this is still the right way to do this?  I just fixed python3 quality.